cisco_secure_email_threat_defense/assets/logs/cisco-secure-email-threat-defense_tests.yaml

```diff
@@ -0,0 +1,367 @@
+      }
+    service: "message"
+    result:
+      custom:
+        action:
+          folder: "trash"
+          isAutoRemediated: false
+          publicApiClientId: ""
+          timestamp: "2024-07-17T10:49:26Z"
+          type: "move"
+        deliveredTo:
+        - "test.user@abc.com"
+        direction: "internal"
+        domain: "abc.com"
+        envelopeTo:
+        - "test.user@abc.com"
+        fromAddress: "rrr@abc.com"
+        http:
+          url:
+          - "http://ihaveabad.dffd/"
+          - "https://sdfsd.cdf/"
+        id: "0cdac345-c111-111c-a978-11162c0da557"
+        internetMessageId: "<AAB-k2pWHpwdBQ0qH1FZGts+AUZyq0w3oMpShp_AAAu9P6JJnhA@mail.gmail.com>"
+        mailboxes:
+        - "test.user@abc.com"
+        - "rrr@abc.com"
+        network:
+          client:
+            geoip: {}
+            ip: "100.100.100.100"
+          destination:
+            geoip: {}
+            ip: "10.10.10.10"
+        returnPath: "rrr@abc.com"
+        senderDomain: "abc.com"
+        senderName: "abc"
+        status: "Info"
+        subject: "verdict override"
+        timestamp: "2024-07-17T05:57:23Z"
+        toAddresses:
+        - "test.user@abc.com"
+        usr:
+          id: "11fa1d11-4f4d-4596-ab02-8dec27d4808e"
+        verdict:
+          businessRisk: ""
+          category: ""
+          isManualVerdict: true
+          isRetroVerdict: false
+          latestVerdict: "neutral"
+          originalVerdict: "malicious"
+          publicApiClientId: ""
+          techniques:
+          -
+            severity: "high"
+            description: "http://ihaveabad.dffd/"
+            type: "Malicious URL"
+          -
+            severity: "medium"
+            description: "Sender domain crestdata.ai rarely communicates with recipient domain onmicrosoft.com"
+            type: "Rare sender domain for recipient domain"
+          -
+            severity: "medium"
+            description: "Message segment 'http://ihaveabad.dffd/ https://sdfsd.cdf/ -- *Disclaimer:* The information contained in this electronic message and any attachments to this message are intended for the exclusive use of the addressee(s) and may contain confidential or privileged information. ' requests user to click a link"
+            type: "Link visit request"
+          -
+            severity: "medium"
+            description: "Message segment 'If you are not the intended recipient, please notify the sender at Crest Data Systems immediately and destroy all copies of this message and any attachments.' contains a request for contact"
+            type: "Request for contact details"
+          -
+            severity: "medium"
+            description: "Sender address rrr@abc.com is rarely seen"
+            type: "Rare sender address"
+          -
+            description: "Email is between internal entities"
+            type: "Internal email"
+          timestamp: "2024-07-17T10:48:19Z"
+      message: |-
```

Comment:
I want to confirm whether we want the entire log content as the message or if there could be a better message attribute we can use

Reply:
@vishalshah-dd 
We do not receive any message attribute or other attributes which provides message details.